doc/user_documentation.rst

```diff
@@ -273,26 +273,37 @@ removed after 40 days of build inactivity.
 Pagure Integration
 ------------------
 
-For any pagure instance (including `src.fedoraproject.org <http://src.fedoraproject.org/>`_), you can setup Copr auto-rebuilding and pr/commit flagging on new changes landing into a pagure repository and its open pull requests.
+For any Pagure instance (including `src.fedoraproject.org <http://src.fedoraproject.org/>`_), you can setup Copr
+auto-rebuilding, triggering build on a PR comment containing ``[copr-build]`` keyword and pr/commit flagging on new
+changes landing into a pagure repository and its open pull requests.
 
 Auto-rebuilding
 ^^^^^^^^^^^^^^^
 
-On the Pagure side, you need to set Fedmsg to 'active' in your project settings (in 'Hooks' section almost at the bottom). For some instances
-(e.g. `src.fedoraproject.org <http://src.fedoraproject.org/>`_), this might already be active by default so you don't even need to perform this step.
+To enable auto-rebuilding on push events, you need to set Fedmsg to 'active' in your project settings on Pagure
+(in 'Hooks' section). For some instances (e.g. `src.fedoraproject.org <http://src.fedoraproject.org/>`_), this
+might already be active by default so you don't even need to perform this step.
 
-In Copr, you need an SCM package definition, which may be as simple as specifying a public clone URL of the remote Pagure repository, see :ref:`scm_ref`
-if you need more detailed settings. Also make sure, "Auto-rebuild" checkbox is checked.
+In Copr, you need an SCM package definition, which may be as simple as specifying a public clone URL of the
+remote Pagure repository, see :ref:`scm_ref` if you need more detailed settings. Also make sure,
+"Auto-rebuild" checkbox is checked.
 
-Now your SCM package will get rebuilt on new commits into the main repo as well as into open PRs.
+Now your SCM package will get rebuilt on new commits into the main repo.
 
-Note that built changes coming from pull requests are not actually placed into the main copr repository. Instead, they are being placed into side repositories
-of the names ``<coprname>:pr:<pr_id>``. ``<pr_id>`` is ID of the pull request opened in Pagure. On Fedora, you can enable the side repository to test the changes with:
+Note that built changes coming from pull requests are not actually placed into the main copr repository.
```

Comment:
s/Note that built changes... /Note that pull request build results are placed into side directories containing pull-request IDs, like `<project_name>:pr:<pull_request_id>`.